serverless/messaging/reference-content/snssqs-overview.mdx

```diff
@@ -52,6 +52,10 @@ Credentials are scoped to the namespace (the bearer of the credentials can acces
   - **Write**: send messages to queues and publish in topics.
   - **Manage**: create or edit resources (such as queues and topics).
 
+## SQS/SNS Queues
+
+After creating a namespace and credentials, you can start to [create queues](/serverless/messaging/how-to/create-manage-queues/) in a namespace. You can choose to create either a Standard or FIFO queue, and tweak parameters such as visibility timeout, message retention period, maximum message size and content based deduplication.
```

Comment:
Great point here with my questions on creating credentials in the console.
Wdyt @JulienVdG ?

Reply:
Oh right, the `you need credential` text probably belong here

Reply:
I reworded